diff --git a/node_modules/react-native/Libraries/Renderer/.DS_Store b/node_modules/react-native/Libraries/Renderer/.DS_Store
new file mode 100644
index 0000000..c91247c
Binary files /dev/null and b/node_modules/react-native/Libraries/Renderer/.DS_Store differ
diff --git a/node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js b/node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js
index 1680f6c..de28a8e 100644
--- a/node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js
+++ b/node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js
@@ -5422,7 +5422,8 @@ function getChildHostContext(parentHostContext, type, rootContainerInstance) {
     type === "RCTMultilineTextInputView" || // iOS
     type === "RCTSinglelineTextInputView" || // iOS
     type === "RCTText" ||
-    type === "RCTVirtualText";
+    type === "RCTVirtualText" ||
+    type === "RCTTextImproved";
 
   if (prevIsInAParentText !== isInAParentText) {
     return {
diff --git a/node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-prod.js b/node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-prod.js
index d1b2a91..e443e22 100644
--- a/node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-prod.js
+++ b/node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-prod.js
@@ -3442,7 +3442,8 @@ function pushHostContext(fiber) {
     "RCTMultilineTextInputView" === JSCompiler_inline_result ||
     "RCTSinglelineTextInputView" === JSCompiler_inline_result ||
     "RCTText" === JSCompiler_inline_result ||
-    "RCTVirtualText" === JSCompiler_inline_result;
+    "RCTVirtualText" === JSCompiler_inline_result ||
+    "RCTTextImproved" === JSCompiler_inline_result;
   JSCompiler_inline_result =
     context.isInAParentText !== JSCompiler_inline_result
       ? { isInAParentText: JSCompiler_inline_result }
